conda_build/build.py

```diff
@@ -60,6 +61,13 @@
 
 import conda_build.noarch_python as noarch_python
 from conda_verify.verify import Verify
+from enum import Enum
+
+
+class FileType(Enum):
+    softlink = "softlink"
+    hardlink = "hardlink"
+    directory = "directory"
```

Comment:
Would it make sense for conda-build to import `LinkType` through `conda-interface` in order to decrease the amount of duplication?
Though if we decide to go that route, `LinkType` will need to be included in the `4.2.x` branch of conda